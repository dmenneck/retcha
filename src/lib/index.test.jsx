import React from 'react'
import renderer from 'react-test-renderer'
import { describe, expect, test } from 'vitest'
import { Retcha, useRetcha } from './index'

describe('Retcha', () => {
  test('Retcha component renders correctly', () => {
    const component = renderer.create(
      <Retcha />
    )

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})