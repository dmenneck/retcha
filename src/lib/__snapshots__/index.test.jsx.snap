// Vitest Snapshot v1

exports[`Retcha > Retcha component renders correctly 1`] = `<div />`;
